sec_certs/sample/fips.py

```diff
@@ -459,7 +459,7 @@ def pandas_tuple(self) -> Tuple:
             self.web_data.module_type,
             self.web_data.level,
             self.web_data.embodiment,
-            self.web_data.date_validation[0] if self.web_data.date_validation else np.nan,
+            self.web_data.validation_history[0] if self.web_data.validation_history else np.nan,
```

Comment:
This should likely have `.date` added after the `[0]` right?

Reply:
Sure, thanks.